icons/openvpn.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>OpenVPN icon</title><path d="M12 .715C5.385.715 0 6.047 0 12.61c0 4.36 2.358 8.154 5.896 10.205l.77-5.076a7.046 7.046 0 0 1-1.846-4.719c0-3.897 3.18-7.076 7.13-7.076 3.948 0 7.126 3.179 7.126 7.076 0 1.847-.717 3.488-1.846 4.77l.77 5.078c3.59-2.051 6-5.899 6-10.258C24 6.047 18.615.715 12 .715zm-.05 8.156a3.786 3.786 0 0 0-3.796 3.795 3.738 3.738 0 0 0 2.461 3.54L9.13 24h5.64l-1.435-7.795c1.385-.564 2.41-1.898 2.41-3.539a3.786 3.786 0 0 0-3.795-3.795z"/></svg>
```

Comment:
Looks like the icon isn't properly centered vertically in the viewport - could you move it up so it's centered?

Reply:
It's should be ok now. Thanks.